### ISSUE TITLE
DatePicker Focus Outline not showing after closing calendar

### ISSUE THREAD

**Issue by timotheeguerin** [microsoft/fluentui#15323]
### Environment Information

- **Package version(s)**: `"@fluentui/react": "7.137.5"`
- **Browser and OS versions**: Edge,Chrome

### Describe the issue:

When navigating to a Date picker using the keyboard it will have the focus outline as expected
![image](https://user-images.githubusercontent.com/1031227/94735643-eec9bc80-031f-11eb-89a7-76b8eed70073.png)
however after opening and closing the calendar(Using Escape) the focus outline doesn't show up even though the focus is still on the input(Pressing enter will reopen the calendar).

This is an accessibility issue.

#### Actual behavior:

Input is not outlined(After closing calendar).
![image](https://user-images.githubusercontent.com/1031227/94735809-26386900-0320-11eb-967f-787080f29c71.png)

#### Expected behavior:

Input should be outline as it is currently focused

### If applicable, please provide a codepen repro:
This can easily be reproduced on the samples page https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority:  Normal

Products/sites affected: (if applicable)

**Comment by paulgildea** [comment]
@timotheeguerin Thanks for the issue! If we root cause the issue would you be willing to submit a PR?

@lorejoh12 FYI - we might need to engage with design on this issue, but I believe can follow the same behavior as the other input types like `Dropdown` which place the primary color focus visual when escaping a `ContextalMenu`

This issue is easier to repo when you set `disableAutoFocus` on the control. 

@JustSlone FYI

**Comment by v-vrusse** [comment]
The accessibility team has dinged our app for this issue as well.  We are currently using `office-ui-fabric-react` 7.155.3 in Chrome / Edge.
